client/src/components/core/ms-image/images.ts

```diff
@@ -57,4 +63,17 @@ const File = {
   Zip,
 };
 
-export { CaretExpand, Device, File, FileImport, LogoIconGradient, LogoRowWhite, PasswordLock, SwapArrows, WavyCaretUp };
+export {
+  CaretExpand,
+  Device,
+  File,
+  FileImport,
+  LogoIconGradient,
+  LogoRowWhite,
+  NoImportDone,
```

Comment:
You export them both in `File` and independantly, which means they can be access as

`import { File } from '@/components/core'; File.NoImportError;`
and as
`import { NoImportError } from '@/components/core';`

Best not to put them in `File` at all.

Reply:
I think it's better to add only independently because `File` concerned only the file icon.